src/main/kotlin/lotto/domain/LottoTicket.kt

```diff
@@ -1,6 +1,6 @@
-package lotto
+package lotto.domain
 
-import lotto.LottoRule.LOTTO_NUMBER_COUNT
+import lotto.domain.LottoRule.LOTTO_NUMBER_COUNT
 
 class LottoTickets(val lottoTickets: List<LottoTicket> = emptyList())
 open class LottoTicket(val numbers: List<LottoNumber>) {
```

Comment:
일급 컬렉션 내부를 공개하지 말고 캡슐화로 구현해 봅시다. 
```suggestion
class LottoTickets(private val lottoTickets: List<LottoTicket> = emptyList())
class LottoTicket(private val numbers: List<LottoNumber>) {
```

Reply:
https://github.com/next-step/kotlin-lotto/pull/367/commits/0795b3a522331fd7a0589f0d5cb2e3646d3d41e1

private으로 수정해보았습니다.